src/components/input-field/input-field.e2e.ts

```diff
@@ -360,3 +354,10 @@ describe('limel-input-field', () => {
 async function createPage(content: string) {
     return newE2EPage({ html: content });
 }
+
+function replaceLabelId(HTML: string) {
+    return HTML.replace(
+        /"a_(\d|[a-f]){8}-(\d|[a-f]){4}-(\d|[a-f]){4}-(\d|[a-f]){4}-(\d|[a-f]){12}"/g,
+        '"tf-input-label"'
+    );
+}
```

Comment:
Why can't you just do it simply like this instead of using regex?
```suggestion
function replaceLabelId(labelId: string) {
    return `${a_labelId}` // add a doc here why we add the prefix
    );
}
```